components/LoadingPage/LoadingPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { CircularProgress, Typography } from "@material-ui/core";
+import React from "react";
+import { useStyles } from "./styles";
```

Comment:
General proposal for @saleor/dx 

Can we do:
```
LandingPage.tsx
LandingPage.styles.ts
```

?

It's FAR better developer experience when I can quick look up for a file with unique name (avoiding index files and other repetitive names)

Reply:
I used the pattern that we use in dashboard, cloud and pretty much all of our frontend project :P

Reply:
Sure, thats why I call entire @saleor/dx to confront this. @saleor/frontend cc